src/CattyTests/Bricks/BrickUserDataTests.swift

```diff
@@ -111,4 +111,62 @@ final class BrickWithUserVariableTests: XCTestCase {
         XCTAssertTrue(addBrick.isVarOrListBeingUsed(userList))
         XCTAssertTrue(deleteBrick.isVarOrListBeingUsed(userList))
     }
+
+    func testUserVariableNoListUsedMultipleVariableBrick() {
+        let userVariable = UserVariable(name: "testName")
+        userVariable.value = "testValue"
+
+        let changeVariableBrick = ChangeVariableBrick()
+        let hideTextBrick = HideTextBrick()
+        let showTextBrick = ShowTextBrick()
+
+        changeVariableBrick.setVariable(userVariable, forLineNumber: 1, andParameterNumber: 1)
+        XCTAssertTrue(changeVariableBrick.isVarOrListBeingUsed(userVariable))
+        XCTAssertFalse(hideTextBrick.isVarOrListBeingUsed(userVariable))
+        XCTAssertFalse(showTextBrick.isVarOrListBeingUsed(userVariable))
+
+        hideTextBrick.setVariable(userVariable, forLineNumber: 1, andParameterNumber: 1)
+        XCTAssertTrue(changeVariableBrick.isVarOrListBeingUsed(userVariable))
+        XCTAssertTrue(hideTextBrick.isVarOrListBeingUsed(userVariable))
+        XCTAssertFalse(showTextBrick.isVarOrListBeingUsed(userVariable))
+
+        showTextBrick.setVariable(userVariable, forLineNumber: 1, andParameterNumber: 1)
+        XCTAssertTrue(changeVariableBrick.isVarOrListBeingUsed(userVariable))
+        XCTAssertTrue(hideTextBrick.isVarOrListBeingUsed(userVariable))
+        XCTAssertTrue(showTextBrick.isVarOrListBeingUsed(userVariable))
+    }
+
+    func testUserVariableNoListForBrickWithMultipleFormula() {
+        let userVariable = UserVariable(name: "testName", isList: false)
```

Comment:
Please add a second variable to the test which should not be used in the Brick (_assert false is being used_) unless added to the Brick as part of a formula (for example instead of the "TestFormula" formula).